CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+### CMakeLists.txt ###
+#
+# Run by CMake to generate project solution or makefiles
+#
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+cmake_policy(SET CMP0048 NEW) # Use new style: version set by project()
+
+# For when we set property CXX_STANDARD.
+set (CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Enable solution folders (in IDEs)
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+# Name the solution folder for targets like INSTALL, PACKAGE, ZERO_CHECK
+set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "Meta Targets")
+
+# Which packages should we generate when installing "package" target
+# or running CPack without specifying a generator
+set(CPACK_GENERATOR  WIX ZIP CACHE STRING "List of generators to build packages with")
```

Comment:
Isn't it Windows specific? Also sometimes later you redefine it here: https://github.com/OpenBoardView/OpenBoardView/pull/24/files#diff-c6410798361ab4228b2112b31d72c95eR60 .